src/components/group.ts

```diff
@@ -52,12 +53,54 @@ export module Components {
     }
 
     public fixedWidth(): boolean {
-      return this.components().every((c) => c.fixedWidth());
+      return this._components().every((c) => c.fixedWidth());
     }
 
     public fixedHeight(): boolean {
-      return this.components().every((c) => c.fixedHeight());
+      return this._components().every((c) => c.fixedHeight());
     }
+
+    /**
+     * @return {Component[]} The Components in this Group.
+     */
+    public components(): Component[] {
+      return this._componentList.slice();
+    }
+
+    protected _components(): Component[] {
+      return this._componentList;
+    }
```

Comment:
The protected version and the public version are so similar yet so different...  This could be easily confusing for extenders